modules.d/01systemd-cryptsetup/parse-crypt.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/sh
```

Comment:
depend on `crypt` and remove the non-systemd part... and remove the systemd part from `90crypt`

Reply:
Module adjusted to depend on crypt and systemd parts related to 90crypt removed.